CONTRIBUTING.md

```diff
@@ -37,17 +37,15 @@ new tests, use the existing tests as a guideline for what they should look like.
 Before you send your pull request, make sure all the tests pass! To run all the
 tests, you'll need:
 
-* [PhantomJS][] version 2.0.0 or higher.
+* [NodeJS]: https://nodejs.org/
 
-[PhantomJS]: http://phantomjs.org/
-
-Once you have these, just run `pub run test`.
+Once you have this, just run `pub run test`.
```

Comment:
This isn't true anymore.

We could replace it with `mono_repo presubmit` perhaps...

Reply:
lol true, I could remove the entire section around testing... (the expectation I think is more to rely on CI than locally run)

Reply:
Removing the section would probably be fine.

Using CI is a more accurate reflection of what we do in practice.

Reply:
I updated it to reference mono_repo as a tool, but not a requirement